leptos_macro/src/lib.rs

```diff
@@ -466,6 +466,11 @@ pub fn template(tokens: TokenStream) -> TokenStream {
 ///    recognizes that a particular tag is a component, not an HTML element. It's important to be aware
 ///    of this when using or importing the component.
 ///
+/// 3. A function with the signature `fn MyComponent() -> impl IntoView` does *not* necessarily *need* to
```

Comment:
@gbj Would you prefer I delete this commit then? This would take spot three. I thought it was worth noting, but can see why it might not belong in the docs.